src/Models/Income.cs

```diff
@@ -0,0 +1,12 @@
+namespace server.Models;
+
+public class Income
+{
+    public float Amount { get; set; }
+
+    public required string Date { get; set; }
+    
+    public string? Comment { get; set; }
+
+    public required string Method { get; set; }
```

Comment:
extract `Method` to a rich model, to make it clear what method means or add a comment, I recommend the first solution.

Reply:
what do you mean by reach model ?